### ISSUE TITLE
Есть повтор в папке modules\system\lang\ru\lang.php

### ISSUE THREAD

**Issue by AlecSPB** [mrmlnc/October-Russian-Language#2]
строка 111:        'sendmail' => 'Sendmail',
строка 120:        'sendmail' => 'Sendmail',

**Comment by mrmlnc** [comment]
Да, к сожалению, есть такая проблема. Я создавал пуллреквест в основной ветке и он был принят, но так как разработка постоянно ведётся эту строку случайно добавили обратно.
